IPython/core/interactiveshell.py

```diff
@@ -2826,6 +3005,8 @@ def run_code(self, code_obj, result=None):
           A compiled code object, to be executed
         result : ExecutionResult, optional
           An object to store exceptions that occur during execution.
+        async :  Bool (Experimental)
```

Comment:
General question: with async the almost-a-keyword, is there any indication that `async` as a variable name should be discouraged?